### PR TITLE
Improve grammar and formatting in the README

### PR DESCRIPTION
**Pull request by Calinou**
This also clarifies the fact that Minetest saves `minetest.conf` when closing Minetest for the first time, rather than when starting it for the first time.

## To do

This PR is Ready for Review.

## How to test

Read it :slightly_smiling_face:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,18 +110,18 @@ Where each location is on each platform:
...
 Worlds can be found as separate folders in: `user/worlds/`
 
-Configuration file:
--------------------
+Configuration file
+------------------
 - Default location:
     `user/minetest.conf`
-- It is created by Minetest when it is ran the first time.
+- It is created by Minetest when closing it for the first time.
```
Comment (SmallJoker) [inline README.md]: Still a bit weird. How about `This file is created by closing Minetest for the first time.`

File: README.md [hunk 2]
```diff
@@ -291,114 +291,112 @@ Library specific options:
...
-* Visual Studio 2017
-* Wix Toolset
+* [Visual Studio 2017](https://visualstudio.microsoft.com/)
+* [WiX Toolset](https://wixtoolset.org/)
 
-In Visual Studio 2017 Installer select "Optional Features" -> "Wix Toolset"
+In the Visual Studio 2017 Installer, select **Optional Features -> WiX Toolset**.
 
-Build the binaries like described above, but make sure you unselect "RUN_IN_PLACE".
+Build the binaries like described above, but make sure you unselect ``RUN_IN_PLACE``.
```
Comment (ClobberXD) [inline README.md]: Double-backticks?

Comment (Calinou) [reply README.md]: Looks like I've done too much reStructuredText recently… :no_mouth:

File: README.md [hunk 3]
```diff
@@ -291,114 +291,112 @@ Library specific options:
...
-`freetype` is optional, it allows true-type font rendering.
 
-`luajit` is optional and replaces the integrated lua interpreter with a faster just in time interpreter
+- `curl` is optional, but required to read the serverlist, `curl[winssl]` is required to use the content store.
+- `openal-soft`, `libvorbis` and `libogg` are optional, but required to use sound.
+- `freetype` is optional, it allows true-type font rendering.
+- `luajit` is optional; it replaces the integrated Lua interpreter with a faster just-in-time interpreter.
 
-There are other libaries that are optional, but are not tested if they can build and linked correctly.
+There are other optional libraries, but are not tested if they can build and linked correctly.
```
Comment (paramat) [inline README.md]: '... and link correctly'?

File: README.md [hunk 4]
```diff
@@ -291,114 +291,112 @@ Library specific options:
...
 
-`--triplet` specify the target triplet e.g. x64-windows x86-windows
+Use `--triplet` specify the target triplet, e.g. `x64-windows` or `x86-windows`.
```
Comment (paramat) [inline README.md]: Use `--triplet` to specify ...

File: README.md [hunk 5]
```diff
@@ -291,114 +291,112 @@ Library specific options:
...
+**Note:** If you don't understand the previous commands, please read the official Docker documentation before use.
 
-You can also host your minetest server inside a Kubernetes cluster. See our example implementation in `misc/kubernetes.yml`.
+You can also host your Minetest server inside a Kubernetes cluster. See our example implementation in [`misc/kubernetes.yml`](misc/kubernetes.yml).
 
 
 Version scheme
 --------------
-We use `major.minor.patch` since 5.0.0-dev. Prior to that we used `0.major.minor`.
+We use `major.minor.patch` since 5.0.0-dev. Prior to that, we used `0.major.minor`.
```
Comment (paramat) [inline README.md]: Added comma not needed.

File: README.md [hunk 6]
```diff
@@ -409,4 +407,4 @@ minor/trivial features considered necessary.
 
 Since 5.0.0-dev and 0.4.17-dev, the dev notation refers to the next release,
 i.e.: 5.0.0-dev is the development version leading to 5.0.0.
-Prior to that we used `previous_version-dev`.
+Prior to that, we used `previous_version-dev`.
```
Comment (paramat) [inline README.md]: As above.

File: README.md (hunk 5 above)
Comment (Calinou) [reply README.md]: I think it sounds much more natural to have a comma here, as "we used…" introduces a different proposition.

**Comment by pauloue** [comment]
You could fix the default pitch move key as well (P, not L).

File: README.md (hunk 5 above)
Comment (paramat) [reply README.md]: I can see why it's tempting, but it's worse grammar. I'm a native English speaker and writer.
Anyway, @Ezhh will know better than either of us =)

File: README.md [hunk 7]
```diff
@@ -291,114 +291,112 @@ Library specific options:
...
 
-### Compiling and Installing the dependencies
+### Compiling and installing the dependencies
 
 It is highly recommended to use vcpkg as package manager.
 
 #### a) Using vcpkg to install dependencies
 
-After you successfully built vcpkg you can easily install the required libaries.
+After you successfully built vcpkg, you can easily install the required libraries:
```
Comment (paramat) [inline README.md]: Unnecessary comma, although it's not a disaster to have one.

File: README.md [hunk 8]
```diff
@@ -291,114 +291,112 @@ Library specific options:
...
 vcpkg install irrlicht zlib curl[winssl] openal-soft libvorbis libogg sqlite3 freetype luajit --triplet x64-windows
 ```
-`curl` is optional, but required to read the serverlist, `curl[winssl]` is required to use the content store.
-
-`openal-soft`, `libvorbis` and `libogg` are optional, but required to use sound.
-
-`freetype` is optional, it allows true-type font rendering.
 
-`luajit` is optional and replaces the integrated lua interpreter with a faster just in time interpreter
+- `curl` is optional, but required to read the serverlist, `curl[winssl]` is required to use the content store.
```
Comment (paramat) [inline README.md]: No comma needed before 'but'.

File: README.md [hunk 9]
```diff
@@ -291,114 +291,112 @@ Library specific options:
...
+- `openal-soft`, `libvorbis` and `libogg` are optional, but required to use sound.
+- `freetype` is optional, it allows true-type font rendering.
+- `luajit` is optional; it replaces the integrated Lua interpreter with a faster just-in-time interpreter.
```
Comment (paramat) [inline README.md]: ';' is inconsistent.

File: README.md [hunk 10]
```diff
@@ -291,114 +291,112 @@ Library specific options:
...
+### Windows Installer using WiX Toolset
 
 Requirements:
-* Visual Studio 2017
-* Wix Toolset
+* [Visual Studio 2017](https://visualstudio.microsoft.com/)
+* [WiX Toolset](https://wixtoolset.org/)
 
-In Visual Studio 2017 Installer select "Optional Features" -> "Wix Toolset"
+In the Visual Studio 2017 Installer, select **Optional Features -> WiX Toolset**.
```
Comment (paramat) [inline README.md]: Unnecessary comma.

**Comment by paramat** [comment]
:-1: there's one very bad usage of a comma that needs fixing.
There are some other dubious usages of commas but those are more debateable so i won't insist on changing those.

**Comment by Calinou** [comment]
@paramat Changes made, please take a look again.

**Comment by paramat** [comment]
Thanks :+1:

**Comment by paramat** [comment]
Merged.
9b067efb99c7b8682639736e7c4908b2e3e6e524

**Pull request closed without merging**